.circleci/config.yml

```diff
@@ -41,7 +42,10 @@ jobs:
       - run:
           name: Run remote verification test against dev (with auth)
           command: bash -i -c 'npm run test-dev-auth'
-
+      - slack/status:
+          fail_only: true
+          failure_message: Smoke tests failed!
+          webhook: https://hooks.slack.com/services/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
```

Comment:
Not sure, but should this be a parameter? Otherwise the webhook could get spammed?

Reply:
Yes, it should be an encrypted parameter

Reply:
I was confused about this because I thought the webhook had authentication so this shouldn't matter. Will redo with the parameter.